modules/database/database.bal

```diff
@@ -562,10 +569,10 @@ public isolated transactional function getTimelineStepList(int experimentId, boo
         query.push(`DATE_FORMAT(start, '%Y-%m-%dT%H:%i:%S') AS start, DATE_FORMAT(end, '%Y-%m-%dT%H:%i:%S') AS end, `);
     }
 
-    query.push(`status, processorName`);
+    query.push(`status, processorName, processorVersion, processorLocation, `);
```

Comment:
you need to be very careful with the commas when building these multiline queries. The last comma is doubled in the next lines. (unfortunately there is no better way...)